From db494ff749971d517ada6a7cb7bde71739efcd6c Mon Sep 17 00:00:00 2001
From: Jianqun xu <jay.xu@rock-chips.com>
Date: Fri, 4 Dec 2015 15:08:58 +0800
Subject: [PATCH] ARM64: dts: rockchip: add compatible for act8846 fit for
 u-boot

Since U-boot only looks up "act,act8846", it's better to add one
more compatible name for 8846, to keep uboot without change.

Change-Id: Ic8935557168c95dd87a38d2f4b7f987b9f0da8c9
Signed-off-by: Jianqun xu <jay.xu@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3368-tb.dts | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3368-tb.dts b/arch/arm64/boot/dts/rockchip/rk3368-tb.dts
index ab70d9eba52a..b562ed1021ff 100644
--- a/arch/arm64/boot/dts/rockchip/rk3368-tb.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3368-tb.dts
@@ -168,7 +168,11 @@
 	};
 
 	act8846: act8846@5a {
-		compatible = "active-semi,act8846";
+		/*
+		 * Note: u-boot ONLY match old compatible,
+		 * it's better to add both instances here.
+		 */
+		compatible = "act,act8846", "active-semi,act8846";
 		reg = <0x5a>;
 		status = "okay";
 		system-power-controller;
-- 
2.35.3

